python/src/ExperimentIntegration_doc.i.in

```diff
@@ -0,0 +1,163 @@
+%feature("docstring") OT::ExperimentIntegration
+"Create a quadrature rule for numerical integration based on a weighted experiment.
+
+Available constructors:
+    ExperimentIntegration(*weightedExperiment*)
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+weightedExperiment : :class:`~openturns.WeightedExperiment`
+    The weighted experimental design.
+
+See also
+--------
+openturns.WeightedExperiment
+
+Notes
+-----
+This class creates an integration method using a weighted experimental
+design.
+It can be used on a function with arbitrary input and
+output.
+
+
+Examples
+--------
+Integrate the Ishigami physical model.
+
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> from openturns.usecases import ishigami_function
+>>> im = ishigami_function.IshigamiModel()
+>>> sampleSize = 100000
+>>> experiment = ot.MonteCarloExperiment(im.distributionX, sampleSize)
+>>> integration = otexp.ExperimentIntegration(experiment)
+>>> approximatedOutputMean = integration.integrate(im.model)
+>>> print(approximatedOutputMean)
+[3.5...]
+
+Compute the L2 norm of a function.
+
+>>> im = ishigami_function.IshigamiModel()
+>>> centeredIshigamiFunction = ot.SymbolicFunction(
+...     ['x1', 'x2', 'x3'],
+...     ['sin(x1) + 7 * (sin(x2)) ^ 2 + 0.1 * x3^4 * sin(x1) - 3.5']
+... )
+>>> sampleSize = 100000
```

Comment:
Actually, this is not as big as it seems.

| n | Approximate integral |
|---|---|
| 1 | 1.94 |
| ... | ... |
| 512 | 3.18 |
| 1024 | 3.61 |
| 2048 | 3.55 |
| 4096 | 3.56 |
| 8192 | 3.46 |
| 16384 | 3.47 |
| 32768 | 3.48 |
| 65536 | 3.49 |
| 131072 | 3.47 |
| 262144 | 3.50 |
| 524288 | 3.50 |

Therefore, rounding to the nearest real number with 1 significant figure after the decimal, the minimum sample size so that the result is 3.5 for any greater sample size is 4096 (up to a factor 2). Keeping a slight margin, I can reduce this number down to 8192.

Reply:
Fixed at feae6a06b.